src/routes/api/available_models/+server.ts

```diff
@@ -0,0 +1,5 @@
+import { models } from "$lib/server/models";
+
+export async function GET() {
+	return Response.json(models);
```

Comment:
I think `models` is meant as a server side var and serializing the whole thing would maybe make it too easy to leak some private properties by accident.

It could be worth mapping `models` to return only the properties you're interested in ? See the [load function](https://github.com/huggingface/chat-ui/blob/41f8b74a10dbb3857ba9b2aa5ab59eb0d0e4e6e6/src/routes/%2Blayout.server.ts#L106-L120) that we use here for example, let me know what you think!

Reply:
make sens. on it.